src/Generator.jl

```diff
@@ -30,13 +30,17 @@ Contents of the default `make.jl` file.
 function make(pkgname; format = :html)
     fmtpkg = format === :markdown ? ", DocumenterMarkdown" :
              format === :pdf      ? ", DocumenterLaTeX" : ""
+    fmtstr = format === :html ? "Documenter.HTML()" :
+             format === :markdown  ? "DocumenterMarkdown.Markdown()" :
```

Comment:
As we're doing `using DocumenterMarkdown/LaTeX`, this should be sufficient:

```suggestion
             format === :markdown  ? "Markdown()" :
```